container/Dockerfile

```diff
@@ -8,6 +8,13 @@ RUN python3 -m venv /venv
 ENV PATH="/venv/bin:$PATH"
 ENV PIP_REQUIRE_VIRTUALENV=true
 
+# Install golang needed by extensions
+ENV GO_VERSION=1.21.0
+ENV PATH="/usr/local/go/bin:${PATH}"
+RUN  wget --progress=dot:giga "https://go.dev/dl/go${GO_VERSION}.linux-$(dpkg --print-architecture).tar.gz" \
```

Comment:
nit: one extra whitespace in front of `wget`.
Maybe `ADD` command instead of `wget`?